ui/app/AppLayouts/Wallet/services/dapps/DAppsRequestHandler.qml

```diff
@@ -484,13 +484,11 @@ SQUtils.QObject {
 
             // Beware, the tx values are standard blockchain hex big number values; the fees values are nim's float64 values, hence the complex conversions
             if (!!tx.maxFeePerGas && !!tx.maxPriorityFeePerGas) {
```

Comment:
as we are ine post eip1559 case, both are present, we do not need to set gas price i believe

Reply:
exactly, that was my question about, [here](https://github.com/status-im/status-desktop/pull/16348#discussion_r1761565396).